html/semantics/forms/form-submission-0/enctypes-helper.js

```diff
@@ -90,12 +126,16 @@
 
   // This function returns a function to add individual form tests corresponding
   // to some enctype.
-  // `expectedBuilder` is a function that takes two parameters: `expected` (the
-  // `expected` value of a test) and `serialized` (the actual form body
-  // submitted by the browser), and returns the correct form body that should
-  // have been submitted. This is necessary in order to account for th
+  // `expectedBuilder` is an optional callback that takes two parameters:
+  // `expected` (the `expected` property passed to a test) and `serialized` (the
+  // actual form body submitted by the browser, isomorphic-decoded). It must
+  // return the correct form body that should have been submitted,
+  // isomorphic-encoded. This is necessary in order to account for the
   // multipart/form-data boundary.
```

Comment:
Somewhere deep in these nested functions the code handles formEncoding and other parameters, but those aren't documented. So I guess what is missing is explaining what kind of function is actually returned.

Reply:
There you go.